packages/expo-doctor/src/checks/ProjectSetupCheck.ts

```diff
@@ -48,9 +51,9 @@ export class ProjectSetupCheck implements DoctorCheck {
 
     if (lockfiles.length > 1) {
       issues.push(
-        `This project has multiple package manager lock files (${lockfiles.join(
+        `Multiple lock files detected (${lockfiles.join(
```

Comment:
- more concise intro
- ensure message isn't just about eas build, since doctor is a tool that is not coupled to eas